resource/test_certs/README.md

```diff
@@ -4,8 +4,10 @@ Contains the following files:
 
 * ca.crt: CA certificate
 * ca.key: CA private key
-* node.crt: server certificate
-* node.key: server private key
+* node.crt: node client certificate
```

Comment:
I'm using the "Common Name" for now as I'm still trying to figure out how I'm going to go from Context.User to filename. I probably does make sense to always have *.client.*, but I'll leave that for when I start loading something other than node.crt